src/Randomizer.SMZ3.Tracking/VoiceCommands/ChatIntegrationModule.cs

```diff
@@ -52,6 +58,11 @@ public ChatIntegrationModule(Tracker tracker, IChatClient chatClient, ILogger<Ch
                 var winningNumber = (int)result.Semantics[WinningGuessKey].Value;
                 await DeclareGanonsTowerGuessingGameWinner(winningNumber);
             });
+
+            AddCommand("Track Content", GetTrackContent(), (tracker, result) =>
+            {
+                var task = AskChatAboutContent();
```

Comment:
Please use async/await here like the others. If you discard a task like this, it might cause exceptions to go unnoticed.